src/main/java/org/sagebionetworks/web/client/DisplayUtils.java

```diff
@@ -256,6 +256,10 @@ public static void notify(String message, NotificationVariant variant) {
 		notify(message, variant, null);
 	}
 
+	public static void notify(String title, String message, NotificationVariant variant) {
+		notify(message, variant, new ToastMessageOptions.Builder().setTitle(title).build());
+	}
```

Comment:
It's common to have a toast with just a title and a message. Handy to have this so we don't have to pull out `ToastMessageOptions.Builder`